controllers/util/storageconsumer.go

```diff
@@ -79,6 +79,20 @@ type StorageConsumerResources interface {
 	ReplaceRbdClientProfileName(string)
 	ReplaceCephFsClientProfileName(string)
 	ReplaceNfsClientProfileName(string)
+
+	// Getters for keys
+	GetRbdRadosNamespaceNameKey() string
+	GetSubVolumeGroupNameKey() string
+	GetSubVolumeGroupRadosNamespaceNameKey() string
+	GetCsiRbdProvisionerCephUserNameKey() string
+	GetCsiRbdNodeCephUserNameKey() string
+	GetCsiCephFsProvisionerCephUserNameKey() string
+	GetCsiCephFsNodeCephUserNameKey() string
+	GetCsiNfsProvisionerCephUserNameKey() string
+	GetCsiNfsNodeCephUserNameKey() string
+	GetRbdClientProfileNameKey() string
+	GetCephFsClientProfileNameKey() string
+	GetNfsClientProfileNameKey() string
```

Comment:
Keys are constant and do not change, and the entire point of this helper struct is to make sure you don't need to deal of care about the keys. Adding "keys accessor" is against the design of this util